docs/topics/whatsnew15.md

```diff
@@ -0,0 +1,740 @@
+
+The property [`Char.category`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/category.html) and its return type
+enum class [`CharCategory`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/-char-category/), which indicates
+a char's general category according to Unicode, are now also available in multiplatform projects.
+
+[Learn more about characters](https://kotlinlang.org/docs/basic-types.html#characters).
+
+### New collections function firstNotNullOf()
+
+The new `firstNotNullOf()`<!--(TODO link)--> and `firstNotNullOfOrNull()`<!--(TODO link) --> functions combine [`mapNotNull()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/map-not-null.html)
+with [`first()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/first.html) or [`firstOrNull()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/first-or-null.html).
+They map the original collection with the custom selector function and return the first non-null value. If there is no such value,
+`firstNotNullOf()` throws an exception, and `firstNotNullOfOrNull()` returns null.
+
+```kotlin
+fun main() {
+//sampleStart
+    val data = listOf("Kotlin", "1.5")
+    println(data.firstNotNullOf(String::toDoubleOrNull))
+    println(data.firstNotNullOfOrNull(String::toIntOrNull))
+//sampleEnd
+}
+```
+
+### Strict version of String?.toBoolean()
+
+Two new functions introduce case-sensitive strict versions of the existing [String?.toBoolean()](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/to-boolean.html):
+* `String.toBooleanStrict()`<!--(TODO link)-->  throws an exception for all inputs except the literals `true` and `false`.
+* `String.toBooleanStrictOrNull()`<!--(TODO link)-->  returns null for all inputs except the literals `true` and `false`.
+
+```kotlin
+fun main() {
+//sampleStart
+    println("true".toBooleanStrict())
+    println("1".toBooleanStrictOrNull())
+    println("1".toBooleanStrict())
+//sampleEnd    
+}
+```
+
+## kotlin-test library
+The [kotlin-test](https://kotlinlang.org/api/latest/kotlin.test/) library introduces some new features:
+* [Simplified test dependencies usage in multiplatform projects](#simplified-test-dependencies-usage-in-multiplatform-projects)
+* [Automatic selection of a testing framework for Kotlin/JVM source sets](#automatic-selection-of-a-testing-framework-for-kotlin-jvm-source-sets)
+* [Assertion function updates](#assertion-function-updates)
+
+### Simplified test dependencies usage in multiplatform projects
+
+Now you can use the `kotlin-test` dependency to add dependencies for testing in the `commonTest` source set, and the Gradle plugin
+will infer the corresponding platform dependencies for each source set:
+* `kotlin-test-junit`/`kotlin-test-junit-5`/`kotlin-test-testng` for JVM source sets, see [automatic choice of a testing framework for Kotlin/JVM source sets](#automatic-selection-of-a-testing-framework-for-kotlin-jvm-source-sets)
```

Comment:
@p7nov to our discussion in another PR:

>Please avoid using / as a conjunction. It's often unclear what you're replacing with it: and, or, something else?

Should I also fix it here?

Reply:
Yes. Let's leave only the default `-junit` in the beginning and explain other options before the link.

Reply:
>explain other options before the link

Can't catch what do you mean. Can discuss in DM.